packages/snaps-sdk/src/types/methods/write-device.ts

```diff
@@ -0,0 +1,40 @@
+import type { Hex } from '@metamask/utils';
+
+import type { ScopedDeviceId } from '../device';
+
+/**
+ * The request parameters for the `snap_writeDevice` method when writing to a
+ * HID device.
+ */
+type HidWriteParams = {
+  /**
+   * The type of the device.
+   */
+  type: 'hid';
+
+  /**
+   * The ID of the device to write to.
+   */
+  id: ScopedDeviceId<'hid'>;
+
+  /**
+   * The data to write to the device.
+   */
+  data: Hex;
```

Comment:
Same question as read with regards to what we want this type to be and what is most performant